src/org/opensolaris/opengrok/authorization/AuthorizationFramework.java

```diff
@@ -145,11 +144,26 @@ public boolean isAllowed(HttpServletRequest request, Project project) {
                 request,
                 "plugin_framework_project_cache",
                 project,
-                new Predicate<IAuthorizationPlugin>() {
+                new AuthorizationEntity.PluginDecisionPredicate() {
             @Override
-            public boolean test(IAuthorizationPlugin plugin) {
+            public boolean decision(IAuthorizationPlugin plugin) {
                 return plugin.isAllowed(request, project);
             }
+        }, new AuthorizationEntity.PluginSkippingPredicate() {
+            @Override
+            public boolean shouldSkip(AuthorizationEntity entity) {
```

Comment:
would be more readable if `entity` was called `authEntity` or such. similarly for the other `isAllowed()` method